arquillian.xml

```diff
@@ -0,0 +1,14 @@
+<arquillian xmlns="http://jboss.org/schema/arquillian"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xsi:schemaLocation="http://jboss.org/schema/arquillian
+    http://jboss.org/schema/arquillian/arquillian_1_0.xsd">
+
+
+    <container qualifier="wildfly" default="true">
+        <configuration>
+            <property name="jbossHome">target/wildfly-8.1.0.Final</property>
```

Comment:
Wildfly version should be property on pom and we should use resource filtering here.